src/content/pt/2019/compression.md

```diff
@@ -0,0 +1,369 @@
+        <td class="numeric">59.28%</td>
+        <td class="numeric">64.54%</td>
+        <td class="numeric">58.26%</td>
+      </tr>
+      <tr>
+        <td><code>gzip</code></td>
+        <td class="numeric">29.33%</td>
+        <td class="numeric">30.20%</td>
+        <td class="numeric">30.87%</td>
+        <td class="numeric">31.22%</td>
+      </tr>
+      <tr>
+        <td><code>br</code></td>
+        <td class="numeric">4.41%</td>
+        <td class="numeric">10.49%</td>
+        <td class="numeric">4.56%</td>
+        <td class="numeric">10.49%</td>
+      </tr>
+      <tr>
+        <td><code>deflate</code></td>
+        <td class="numeric">0.02%</td>
+        <td class="numeric">0.01%</td>
+        <td class="numeric">0.02%</td>
+        <td class="numeric">0.01%</td>
+      </tr>
+      <tr>
+        <td><em>Outro / Inválido</em></td>
+        <td class="numeric">0.01%</td>
+        <td class="numeric">0.02%</td>
+        <td class="numeric">0.01%</td>
+        <td class="numeric">0.02%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Compressão de conteúdo próprio versus terceiros por tipo de dispositivo.") }}</figcaption>
+</figure>
+
+## Como identificar oportunidades para compressão
+
+A ferramenta [Lighthouse](https://developers.google.com/web/tools/lighthouse) da Google permite aos utilizadores correrem uma série de auditorias contra páginas web. A [auditoria de compressão de texto](https://developers.google.com/web/tools/lighthouse/audits/text-compression) avalia se o site pode beneficiar de compressão baseada em texto adicional. Isso é feito através duma tentativa de compressão dos recursos avaliando se o tamanho dos objectos pode ser reduzido pelo menos 10% e 1400 bytes. Dependendo da pontuação, é possível ver uma recomendação para compressão nos resultados, com uma lista dos recursos específicos que podem ser comprimidos. 
+
+{{ figure_markup(
+  image="ch15_fig8_lighthouse.jpg",
+  caption="Sugestões de compressão do Lighthouse.",
+  description="A captura dum relatório do Lighthouse que mostra a lista de recursos (com respectivos nomes) e o tamanho do potencial ganho. Para o primeiro elemento há um ganho potencial significativo de 91 KB para 73 KB, enquanto que para outros ficheiros mais pequenos de 6 KB ou menos, encontram-se ganhos menores de 4 KB para 1 KB.",
+  width=600,
+  height=303
+  )
+}}
+
+Devido ao facto do [HTTP Archive correr auditorias Lighthouse](./methodology#lighthouse) para cada página mobile, podemos agregar as pontuações para todos os sites para entender que mais oportunidade existem para comprimir mais conteúdo. No total, 62% dos sites passam esta auditoria e quase 23% têm uma pontuação abaixo de 40. Isto significa que mais de 1.2 milhões de sites poderiam beneficiar com uma compressão baseada em texto adicional.
+
+{{ figure_markup(
+  image="fig11.png",
+  caption='Pontuações da auditoria de "compressão de texto" do Lighthouse.',
```

Comment:
Not sure what to do here as the audit is called "enable text compression" in Lighthouse so should we translate or not? Just checked and the other translations did translate it so probably ok that Portuguese does too. What do you think?

Reply:
I missed the "enable" part which in portuguese could be something like "activar".